lib/ui/shared/const_app_colors.dart

```diff
@@ -50,4 +50,8 @@ const Color authBtnColor = Color(0xff363636);
 const Color preferenceBorderblack = Color(0xFF323232);
 
 //status colors
-const Color clearStatusBtnBorderColor = Color(0xffa1a1a1);
\ No newline at end of file
+const Color clearStatusBtnBorderColor = Color(0xffa1a1a1);
+
+//Preferences colors
+const Color kcPrefTitleColor = Color.fromRGBO(29, 29, 29, 1);
```

Comment:
Why is preferences having it's own color, is the app colors not general

Reply:
done